### PR TITLE
docs: Create CODE_OF_CONDUCT.md

### LINKED ISSUE #48: docs: Code of Conduct
The project is not having code of conducts so can I wor on it @s-ayush2903

### PR DESCRIPTION
**Pull request by Tejas1510**
I Have added a formal Code of Conduct File which Fixes #48

### CODE REVIEW

**Comment by Tejas1510** [comment]
@s-ayush2903 Please Give It a look

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```
Comment (s-ayush2903) [inline CODE_OF_CONDUCT.md]: We don't need it

**Comment by Tejas1510** [comment]
Ok i will remove it @s-ayush2903

**Comment by Tejas1510** [comment]
@s-ayush2903 I have removed the part you asked for and have made a PR

**Comment by s-ayush2903** [comment]
Mention the issue in the description which this PR solves via using `Fixes #issue_number`, after doing so, when the PR is merged, the issue will be closed automatically.

**Comment by Tejas1510** [comment]
I have added the Code of Conduct file which Fixes #48

**Comment by Tejas1510** [comment]
@s-ayush2903 I have now added the issue number in the description 
Please give it a lool

**Comment by s-ayush2903** [comment]
Nope, it stiil isn't correct, you are supposed to add it in issue description, i.e., [here](https://github.com/s-ayush2903/KeepItClean/pull/49#issue-533066856)

**Comment by Tejas1510** [comment]
@s-ayush2903 is it correct know

**Comment by s-ayush2903** [comment]
Yes, fine. Btw you're supposed to add it in the very first line of issue desc. and then the description follows; no worries, it is fine now

**Comment by Tejas1510** [comment]
Thanks a lot @s-ayush2903 for guiding me
I am new to it and learning
Definitely will not repeat it next time

**Review by s-ayush2903 — APPROVED** [review]
Good work, merging! :tada:

**Pull request merged**